Makefile

```diff
@@ -9,3 +9,5 @@ webserver:
 clean:
 	rm -rf *.o webserver
 
+integrate:
```

Comment:
Thank you, will probably have to merge with my makefile in the master now tho just so you know.